MangoPay/KycDocument.php

```diff
@@ -8,18 +8,33 @@
 class KycDocument extends Libraries\Document
 {
     /**
+     * The user ID who owns this document
      * @var string
      * @see \MangoPay\KycDocumentType
      */
     public $Type;
 
+	/**
+	 * The label of KYC document type
+	 * @var string
+	 */
+	public $TypeLabel;
+
     /**
+     * The status of the KYC document
      * @var string
      * @see \MangoPay\KycDocumentStatus
      */
     public $Status;
 
+	/**
+	 * The label of KYC document status
+	 * @var string
+	 */
+	public $StatusLabel;
```

Comment:
the code is not well alligned

Reply:
How can we ensure proper alignment?

Reply:
You are using tabs instead of spaces, maybe add this to the code fixer

Reply:
I've addressed the alignment issue. Please review it again when you have time.